main.go

```diff
@@ -146,6 +148,40 @@ func rm(c config.Config, path, name string) {
 	c.Save(path)
 }
 
+func update(c config.Config, path, name string) {
+	_, err := c.Get(name)
+	if err != nil {
+		fmt.Println("It's not posible update "+name+" as it doesn't exist.")
+		return
+	}
+
+	search := c.Position(name)
```

Comment:
Instead of `c.Postion(name)` and the loop below you can simply use `cmd, err := c.Get(name)`, so you reuse all the logic on the `Config`.

Once you have the command (cmd) is time to convert it on a valid string (the way is stored on the file). This logic is already implemented on `Config.toLines()`. However is not accessible, so we will need to do a refactor there. Ideally I would like to see something like this:

```go
// Get the command you want to update
currentAlias, err := c.Get(name)
if err != nil {...}

// Dispay the editor with the command on it to capture the input
newAlias, err := editor.CaptureInputFromEditor(
    editor.GetPreferredEditorFromEnvironment,
    currentAlias.String(), // This is where you have to move the toLines() logic
)
if err != nil {...}

// Replace existing command
err = c.Update(name, newAlias)
if err != nil {...}

// Save the configuration to a file
c.Save()
```

We can do some pair programming if you like :)